data/tree/equality tester/47bd9ed4-f7b7-4872-ab54-9136ac251f4a.json

```diff
@@ -121,14 +121,14 @@
   "interface": {
     "inputs": {
       "2a9ca66c-22e3-4478-b06b-c1da6e93eeac": {
-        "type": "dictionary",
+        "type": "{string: any}",
         "name": "dict",
-        "description": "The dictionary to test"
+        "description": "The dictionary to test for equality with `reference`"
       },
       "3878ff33-ee0b-4ec7-a9aa-76ea242941bf": {
-        "type": "dictionary",
+        "type": "{string: any}",
```

Comment:
Type should be the same for all trees. Or can maybe rather a reference to `tree` port (to be renamed from `dict`) as we expect the same type: ``` typeof `tree` ```